test/sequential/test-perf-hooks.js

```diff
@@ -1,84 +1,165 @@
-function checkNodeTiming(props) {
-  console.log(props);
-
-  for (const prop of Object.keys(props)) {
-    if (props[prop].around !== undefined) {
-      assert.strictEqual(typeof performance.nodeTiming[prop], 'number');
-      const delta = performance.nodeTiming[prop] - props[prop].around;
-      assert(
-        Math.abs(delta) < (props[prop].delay || delay),
-        `${prop}: ${Math.abs(delta)} >= ${props[prop].delay || delay}`
-      );
-    } else {
-      assert.strictEqual(performance.nodeTiming[prop], props[prop],
-                         `mismatch for performance property ${prop}: ` +
-                         `${performance.nodeTiming[prop]} vs ${props[prop]}`);
-    }
-  }
+// Copy all the values from the getters.
+const initialTiming = { ...performance.nodeTiming };
+
+{
+  const {
+    startTime,
+    nodeStart,
+    v8Start,
+    environment,
+    bootstrapComplete,
+  } = initialTiming;
+
+  assert.strictEqual(startTime, 0);
+  assert.strictEqual(typeof nodeStart, 'number');
+  assert(nodeStart > 0, `nodeStart ${nodeStart} <= 0`);
+  // The whole process starts before this test starts.
+  assert(nodeStart < testStartTime,
+         `nodeStart ${nodeStart} >= ${testStartTime}`);
+
+  assert.strictEqual(typeof v8Start, 'number');
+  assert(v8Start > 0, `v8Start ${v8Start} <= 0`);
+  // V8 starts after the process starts.
+  assert(v8Start > nodeStart, `v8Start ${v8Start} <= ${nodeStart}`);
+  // V8 starts before this test starts.
+  assert(v8Start < testStartTime,
+         `v8Start ${v8Start} >= ${testStartTime}`);
+
+  assert.strictEqual(typeof environment, 'number');
+  assert(environment > 0, `environment ${environment} <= 0`);
+  // Environment starts after V8 starts.
+  assert(environment > v8Start,
+         `environment ${environment} <= ${v8Start}`);
+  // Environment starts before this test starts.
+  assert(environment < testStartTime,
+         `environment ${environment} >= ${testStartTime}`);
+
+  assert.strictEqual(typeof bootstrapComplete, 'number');
+  assert(bootstrapComplete > 0, `bootstrapComplete ${bootstrapComplete} <= 0`);
+  // Bootstrap completes after environment starts.
+  assert(bootstrapComplete > environment,
+         `bootstrapComplete ${bootstrapComplete} <= ${environment}`);
+  // Bootstrap completes before this test starts.
+  assert(bootstrapComplete < testStartTime,
+         `bootstrapComplete ${bootstrapComplete} >= ${testStartTime}`);
 }
 
-checkNodeTiming({
-  name: 'node',
-  entryType: 'node',
-  startTime: 0,
-  duration: { around: performance.now() },
-  nodeStart: { around: 0 },
-  v8Start: { around: 0 },
-  bootstrapComplete: { around: inited, delay: 2500 },
-  environment: { around: 0 },
-  loopStart: -1,
-  loopExit: -1
-});
+function checkNodeTiming(timing) {
+  // Calculate the difference between now() and duration as soon as possible.
+  const now = performance.now();
+  const delta = Math.abs(now - timing.duration);
+
+  log(JSON.stringify(timing, null, 2));
```

Comment:
Rookie question i guess! but why do we log these? for debugging purposes? or something more?